Core/ControlMapper.cpp

```diff
@@ -280,8 +297,12 @@ bool ControlMapper::UpdatePSPState(const InputMapping &changedMapping) {
 		// that still works, though a bit weaker. We could also zero here, but you never know who relies on such strange tricks..
 		// Note: This is an old problem, it didn't appear with the refactoring.
 		if (!changedMapping.IsAxis()) {
-			for (auto &mapping : inputMappings) {
-				if (mapping.IsAxis()) {
+			for (auto &multiMapping : inputMappings) {
+				bool anyAxis = false;
+				for (auto &mapping : multiMapping.mappings) {
+					if (mapping.IsAxis()) {
+						anyAxis = true;
+					}
```

Comment:
Was this meant to be used somewhere?

-[Unknown]

Reply:
Hm, not sure what I did there, from some earlier iteration maybe.. this has gone through a couple of rebases.